pkg/cli/version_test.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright2020TheKubernetesAuthors.
+
+LicensedundertheApacheLicense,Version2.0(the"License");
+youmaynotusethisfileexceptincompliancewiththeLicense.
+YoumayobtainacopyoftheLicenseat
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unlessrequiredbyapplicablelaworagreedtoinwriting,software
+distributedundertheLicenseisdistributedonan"ASIS"BASIS,
+WITHOUTWARRANTIESORCONDITIONSOFANYKIND,eitherexpressorimplied.
+SeetheLicenseforthespecificlanguagegoverningpermissionsand
+limitationsundertheLicense.
+*/
+
+package cli
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("CLI", func() {
+	var (
+		c   *CLI
+		err error
+	)
+
+	BeforeEach(func() {
+		c = &CLI{}
+	})
+
+	Context("newVersionCmd", func() {
+		It("Test the version", func() {
+			cmd := c.newVersionCmd()
+			if err := cmd.Execute(); err != nil {
```

Comment:
We should also be testing the command outputs (ie)

```go
Expect(cmd.Use).NotTo(BeNil())
Expect(cmd.Use).To(ContainSubstring("version"))
```
So on and so forth for other fields.

Refer here for examples: https://github.com/operator-framework/operator-sdk/blob/master/internal/cmd/helm-operator/version/cmd_test.go